tests/rp_cloud_cleanup.py

```diff
@@ -196,11 +196,10 @@ def _log_deleted(_msg):
                     return False
                 # Uninstall/Install plugin
                 try:
-                    _message += "| update plugin "
+                    _message += "| clean plugin "
                     self.utils.rpk_plugin_uninstall('byoc')
```

Comment:
```suggestion
```

i think we should also skip uninstall of the `byoc` plugin if we don't install it as well

Reply:
Agreed

Reply:
But there is originally `byoc-mock` plugin installed by packer with no version command. It would be needed to check if this would work fresh out of the packer.